packages/wallet-service/tests/txProcessor.test.ts

```diff
@@ -515,300 +93,3 @@ describe('NFT metadata updating', () => {
-  });
-  block.tx_id = 'txId6';
-  block.timestamp += 10;
-  block.height = 6;
-  await expect(() => txProcessor.onNewTxEvent(evt)).rejects.toThrow('unlock-utxos-error');
-
-  latestHeight = await getLatestHeight(mysql);
-  expect(latestHeight).toBe(block.height - 1);
-
-  // finally, test the balances
-  await expect(checkUtxoTable(mysql, 5, 'txId2', 0, '00', 'address1', blockReward, 0, null, 2 + blockRewardLock, false)).resolves.toBe(true);
-  await expect(checkUtxoTable(mysql, 5, 'txId3', 0, '00', 'address1', blockReward, 0, null, 3 + blockRewardLock, false)).resolves.toBe(true);
-  await expect(checkUtxoTable(mysql, 5, 'txId4', 0, '00', 'address1', blockReward, 0, null, 4 + blockRewardLock, false)).resolves.toBe(true);
-  await expect(checkUtxoTable(mysql, 5, 'txId5', 0, '00', 'address1', blockReward, 0, null, 5 + blockRewardLock, true)).resolves.toBe(true);
-  await expect(checkAddressTable(mysql, 1, 'address1', null, null, 5)).resolves.toBe(true);
-  // txId5 is locked, so our address balance will be 25600
-  await expect(checkAddressBalanceTable(mysql, 1, 'address1', '00', blockReward * 4, blockReward, null, 5)).resolves.toBe(true);
-});
-
-test.skip('txProcess onNewTxRequest with push notification', async () => {
-  expect.hasAssertions();
-
-  const fakeEvent = JSON.parse(JSON.stringify(eventTemplate)).Records[0];
-  const fakeContext = {
-    awsRequestId: 'requestId',
-  } as unknown as Context;
-  const fakeWalletBalanceValue = { 123: { txId: 'txId' } } as unknown as StringMap<WalletBalanceValue>;
-
-  const addNewTxMock = jest.spyOn(txProcessor, 'addNewTx');
-  const isTransactionNFTCreationMock = jest.spyOn(NftUtils, 'isTransactionNFTCreation');
-  const isPushNotificationEnabledMock = jest.spyOn(pushNotificationUtils, 'isPushNotificationEnabled');
-  const getWalletBalancesForTxMock = jest.spyOn(commons, 'getWalletBalancesForTx');
-  const invokeOnTxPushNotificationRequestedLambdaMock = jest.spyOn(pushNotificationUtils.PushNotificationUtils, 'invokeOnTxPushNotificationRequestedLambda');
-
-  /**
-   * Push notification disabled
-   */
-  addNewTxMock.mockImplementation(() => Promise.resolve());
-  isTransactionNFTCreationMock.mockReturnValue(false);
-  isPushNotificationEnabledMock.mockReturnValue(false);
-
-  await txProcessor.onNewTxRequest(fakeEvent, fakeContext, null);
-
-  expect(invokeOnTxPushNotificationRequestedLambdaMock).toHaveBeenCalledTimes(0);
-
-  /**
-   * Push notification enabled
-   */
-  isPushNotificationEnabledMock.mockReturnValue(true);
-  // Get a valid wallet balance value to invoke push notification lambda
-  getWalletBalancesForTxMock.mockResolvedValue(fakeWalletBalanceValue);
-  invokeOnTxPushNotificationRequestedLambdaMock.mockResolvedValue();
-
-  await txProcessor.onNewTxRequest(fakeEvent, fakeContext, null);
-
-  expect(invokeOnTxPushNotificationRequestedLambdaMock).toHaveBeenCalledTimes(1);
-});
-
-test.skip('onNewTxRequest should send alert on SQS on failure', async () => {
```

Comment:
We are dealing with events differently now, but we're not sending an alert with rich metadata when the daemon stops. We should refactor the daemon to add an alert directly to SQS (instead of just logging the error) and add a test for it.